addons/web/static/src/legacy/js/widgets/colorpicker.js

```diff
@@ -44,7 +44,16 @@ var ColorpickerWidget = Widget.extend({
         // anywhere on the screen, crossing iframes).
         // TODO adapt in master: these events should probably be bound in
         // `start` instead of `init` (at least to be more conventional).
-        this.$documents = $([window.top, ...Array.from(window.top.frames)].map(w => w.document));
+        this.$documents = $([window.top, ...Array.from(window.top.frames)].filter(frame => {
+            try {
+                const document = frame.document;
+                return !!document;
+            } catch (error) {
+                // We cannot access the document (cross origin) or there is no
+                // document.
+                return false;
```

Comment:
Nitpicking: your comment says "or there is no document" but if you reach this code, there is a document which could not be accessed :wink: You can just remove "or there is no document" IMO. Your `return !!document` above is enough to handle the strange case where there would be no document.